charts/agent-control/templates/install.yaml

```diff
@@ -18,8 +18,13 @@ spec:
       serviceAccountName: {{ include "newrelic.common.naming.truncateToDNSWithSuffix" (dict "name" .Release.Name "suffix" "install-job") }}
       containers:
         - name: install-agent-control-deployment
+          {{- if (index .Values "agent-control-deployment").installerImage }}
           image: {{ include "newrelic.common.images.image" ( dict "imageRoot" (index .Values "agent-control-deployment").installerImage "context" .) }}
           imagePullPolicy: {{ (index .Values "agent-control-deployment").installerImage.pullPolicy }}
+          {{- else }}
+          image: "newrelic/newrelic-agent-control-cli:nightly"
+          imagePullPolicy: "IfNotPresent"
+          {{- end }}
```

Comment:
Workaround for "ct issue". Remove before merging once tests work. Or let it there and remove in a future release.